Nodejs/Product/TestAdapter/TestFrameworks/Jasmine/jasmine.js

```diff
@@ -222,14 +222,21 @@ function run_tests(testCases, callback) {
     try {
         var jasmineInstance = initializeJasmine(Jasmine, projectFolder);
         jasmineInstance.configureDefaultReporter({ showColors: false });
-        jasmineInstance.env.specFilter = (spec) => {
-            return testNameList.hasOwnProperty(spec.getSpecName(spec));
-        };
+        setSpecFilter(jasmineInstance, spec => testNameList.hasOwnProperty(spec.getSpecName(spec)));
         jasmineInstance.addReporter(createCustomReporter(callback));
         jasmineInstance.execute(testFileList);
     }
     catch (ex) {
         logError("Execute test error:", ex);
     }
 }
+
+function setSpecFilter(jasmineInstance, specFilter) {
+    if (jasmineInstance.env.configure) {
+        jasmineInstance.env.configure({ specFilter });
```

Comment:
Does this supplement the existing configuration or replace it?

Reply:
It supplements; in particular, it just updates that one property on the existing configuration.